src/components/ProductDetailAddToCart/ProductDetailAddToCart.js

```diff
@@ -116,14 +117,9 @@ export default class ProductDetailAddToCart extends Component {
 
   handleOnClick = () => {
     const { uiStore } = this.props;
-    // This function currently does nothing. When our GraphQL endpoints are available, we'll use them to add the items to the cart.
-    // To test that this is working, uncomment the following lines and check to see that the data is correct
 
-    // const { variantId } = this.props;
-    // const { addToCartQuantity } = this.state;
-
-    // console.log("ID to add to cart: ", variantId);
-    // console.log("Quantity to add to cart: ", addToCartQuantity);
+    // Pass chosen quantity to onClick callback
+    this.props.onClick(this.state.addToCartQuantity);
```

Comment:
Should this be included in the destructure above instead of using `this.props` here?